rs/backend/src/perf.rs

```diff
@@ -36,6 +36,9 @@ impl PerformanceCount {
 pub struct PerformanceCounts {
     pub instruction_counts: VecDeque<PerformanceCount>,
     pub exceptional_transactions: Option<VecDeque<u64>>,
+    // TODO: Delete this once the stable memory migration is complete.  This is used purely to get
```

Comment:
Would it make sense to refer to a Jira ticket to track the follow-up work?

```suggestion
    // TODO[NNS1-ABCD]: Delete this once the stable memory migration is complete.  This is used purely to get
```

Reply:
Sure.  I Assume NNS1 is the preferred prefix?

Reply:
@aterga  I'm not sure I can.  When, in Jira, I try to add a ticket, NNS is not offered as an option in the list of projects.  If you create a ticket, I can add it.  There are various TODOs in the codebase.  It might make sense to go through all of them and assign NNS jira tickets to them.

Reply:
@aterga  I'm not sure I can.  When, in Jira, I try to add a ticket, NNS is not offered as an option in the list of projects.  If you create a ticket, I can add it.  There are various TODOs in the codebase.  It might make sense to go through all of them and assign NNS jira tickets to them.

Reply:
I will mark this conversation as resolved so that this PR can be merged for today's release but am happy to discuss further or make another PR if needed.  Ditto with later conversations.